scripts/build/build.sh

```diff
@@ -21,6 +23,26 @@ BUILD_BACKEND=1
 BUILD_FRONTEND=1
 BUILD_PACKAGE=1
 
+
+function reportFrontEndBuildTime() {
+  targetTag=";target=oss"
+  gitTag=""
+  gitBranch=""
+
+  if `echo "$EXTRA_OPTS" | grep -q enterprise` ; then
+    targetTag=";target=enterprise"
+  fi
+  if [ "$CIRCLE_TAG" != "" ]; then
+    gitTag=";gitTag=${CIRCLE_TAG}"
+  fi
+  if [ "$CIRCLE_BRANCH" != "" ]; then
+    gitBranch=";branch=${CIRCLE_BRANCH}"
+  fi
+
+  exit_if_fail ./scripts/ci-metrics-publisher.sh "grafana.ci-performance.frontend-build\${gitBranch}\${gitTag}\${targetTag}=$1"
+}
```

Comment:
is there a way to move this a helper script so it's not duplicated between build & build-all ? and do we need this for PRs it will create a lot of metrics that won't be super interesting.

Reply:
@torkelo for now i'm only trying to understand graphite ;) But I agree tracking every PR doesn not make much sense, only master is what is interesting.

Reply:
Same with the tags - just trying to figure those out. So let's assume i send a metric like this to Graphite:

`grafana.ci-performance.frontend-build;target=oss`

Should the tag be part of the metric name? When I browse this in play.grafana now I get sth like this where tags are part of the metric name:

![image](https://user-images.githubusercontent.com/2376619/68423958-51afe380-01a3-11ea-999f-d0cebb5c4b12.png)

^^ This is probably me misunderstanding Graphite though ;)

Reply:
@torkelo updated to collect only on master (build job only)